src/base/install/getFilesInDir.m

```diff
@@ -140,7 +140,7 @@
     files = files(cellfun(@(x) isempty(regexp(x,regexptranslate('escape',filesep))),files));
 end
 
-files = strcat(absPath, filesep, files);
+files = strcat(repmat({absPath}, 1, length(files)), repmat({filesep}, 1, length(files)),files);
```

Comment:
In which matlab version does this fail?
Just out of curiosity

Reply:
This came from the forum. As this is equivalent, I though of adding it.